[2025-04-21 12:36:40,862] 23 root - INFO - Enter the try block
[2025-04-21 12:36:40,862] 27 root - INFO - Enter the exception block
[2025-04-21 12:36:40,863] 13 root - ERROR - Error occured in python file [C:\Users\bhara\OneDrive\Desktop\MLops\networksecurity\exception\exception.py] line number [24] error message [division by zero].
